### ISSUE TITLE
[BUG] CentOS Stream 9 libvirt box URL returns 404

### ISSUE THREAD

**Issue by mohd-akram** [EuroLinux/cloud-images-rfc#8]
**Describe the bug**
Unable to install the CentOS Stream 9 Vagrant libvirt box.

**Cloud Image**
9.0.21
Vagrant

**To Reproduce**

1. Create Vagrantfile:

    ```ruby
    Vagrant.configure("2") do |config|
      config.vm.box = "eurolinux-vagrant/centos-stream-9"
    end
    ```

2. Run `vagrant up`

**Expected behavior**
It works.

**Actual behavior**

```
Bringing machine 'default' up with 'libvirt' provider...
==> default: Box 'eurolinux-vagrant/centos-stream-9' could not be found. Attempting to find and install...
    default: Box Provider: libvirt
    default: Box Version: >= 0
==> default: Loading metadata for box 'eurolinux-vagrant/centos-stream-9'
    default: URL: https://vagrantcloud.com/eurolinux-vagrant/centos-stream-9
==> default: Adding box 'eurolinux-vagrant/centos-stream-9' (v9.0.21) for provider: libvirt
    default: Downloading: https://vagrantcloud.com/eurolinux-vagrant/boxes/centos-stream-9/versions/9.0.21/providers/libvirt.box
An error occurred while downloading the remote file. The error
message, if any, is reproduced below. Please fix this error and try
again.

The requested URL returned error: 404
```

**Comment by AlexBaranowski** [comment]
Hi!

Thanks for the great bug report! It is reproducible.

I checked if any other box also has a similar issue - it's quite easy as the buggy box doesn't provide size.
 ![c9-stream](https://user-images.githubusercontent.com/8447941/205940385-b4738497-ff51-428e-953b-b43f06ad9025.png) - and fortunately only this one has this issue.

As our boxes are uploaded by build scripts in our CI/CD, I checked logs and there was a closed connection, so the box wasn't fully uploaded.

I also started the new build for this box. I will notify You when it will be ready (about 1-2 hours).
